docs/upgrade/1.0.0_to_1.1.0.rst

```diff
@@ -0,0 +1,167 @@
+to create a fresh Tails drive on a computer running Windows, Mac, or Linux.
+
+Boot the *Tails 4 Primary USB* on the air-gapped computer you use as the *Secure
+Viewing Station*, and follow the instructions for `manually upgrading from
+another Tails <https://tails.boum.org/upgrade/clone/index.en.html>`__
+to upgrade each workstation USB in turn. This procedure preserves the persistent
+storage volume of each USB drive you upgrade to Tails 4.
+
+Boot each workstation into Tails 4 to verify that the upgrade was successful. On
+the *Admin* and *Journalist Workstations*, set an administrator password on the
+Tails welcome screen, and update the SecureDrop environment using the following
+commands: ::
+
+  cd ~/Persistent/securedrop
+ ./securedrop-admin setup
+ ./securedrop-admin tailsconfig
+
+.. important::
+
+   Until you run these commands, the SecureDrop shortcuts on the Tails desktop
+   will not work, and the graphical updater will no longer report available
+   updates for the SecureDrop code on your workstation.
+
+No additional configuration is required for the *Secure Viewing Station*.
+
+If you experience difficulties with this upgrade, please do not hesitate to
+contact us using any of the methods below. If the upgrade failed and you need
+to restore from a backup, see our :ref:`guide for restoring workstations <restore_workstations>`.
+Make sure you restore to a Tails drive using Tails 3.16 before attempting
+another upgrade to Tails 4.
+
+Troubleshooting ``securedrop-admin``
+------------------------------------
+
+This release migrates the ``securedrop-admin`` command to Python 3, which is
+necessary because the Python 2 series reaches end-of-life on January 1, 2020.
+The ``securedrop-admin setup`` step normally should take care of all the
+required changes.
+
+If you see error messages when running ``securedrop-admin`` or
+``securedrop-admin setup``, we recommend the following:
+
+- Ensure that you are running version 1.1.0 of SecureDrop on your workstation,
+  by running the command ``git status`` in your ``~/Persistent/securedrop``
+  directory. If the output is not "HEAD detached at 1.1.0", perform a
+  :ref:`manual upgrade to SecureDrop 1.1.0 on your workstation <manual_upgrade_to_110>`.
+
+- Check your network connection on your workstation. The
+  ``securedrop-admin setup`` command requires a working Tor connection.
+
+- Make sure that you have `set an admin password <https://tails.boum.org/doc/first_steps/startup_options/administration_password/>`__
+  on the Tails welcome screen.
+
+If this does not solve the problem, clear out any existing Python environments
+on your workstation by following these steps:
+
+- Change into the directory ``~/Persistent/securedrop/admin``.
+
+- Run the following commands from within this directory (make sure to type them
+  exactly as shown, as this is a destructive operation):
+
+.. code:: bash
+
+  rm -rf .venv
```

Comment:
the `f` parameter was not necessary in my local testing

Reply:
dangerous force of habit -- fixed :)